packages/editor-ui/src/plugins/icons.ts

```diff
@@ -102,6 +103,7 @@ addIcon(faArrowLeft);
 addIcon(faArrowRight);
 addIcon(faAt);
 addIcon(faBook);
+addIcon(faBoxOpen);
```

Comment:
do you also need to remove the icon we added?

Reply:
In the latest push the faShapes icon is removed.